Firestore/core/src/api/firestore.h

```diff
@@ -95,6 +95,12 @@ class Firestore : public std::enable_shared_from_this<Firestore> {
   void EnableNetwork(util::StatusCallback callback);
   void DisableNetwork(util::StatusCallback callback);
 
+  /**
+   * Sets the language of the public API in the format of
+   * "gl-<language>/<version>", e.g. `gl-objc/2.0`.
+   */
+  static void SetClientLanguageHeader(std::string language_header);
```

Comment:
This value isn't really a header--it's one of many values contained in a header. However, `SetClientLanguageHeaderValue` seems overblown. What about `SetClientLanguage`?

Reply:
Looking at the original doc, things like `gl-objc` are sometimes called tokens, but I'm not sure `SetClientLanguageToken` is necessarily an improvement. Changed to `SetClientLanguage`.